resources/documentation/stories.rst

```diff
@@ -0,0 +1,467 @@
+=======
+Stories
+=======
+
+This part contains few stories to help you face common tasks / issues seen
+while using the stack.
+
+Management and main configuration
+=================================
+
+Versioning the inventory
+------------------------
+
+It may be a good idea to version the whole /etc/bluebanquise/inventory/
+folder, to keep track of all changes made, and be able to revert in case of
+issues.
+
+.. note::
+  If used, it may be also a good idea to version the
+  /etc/bluebanquise/playbooks/ and /etc/bluebanquise/roles/custom/ folders.
```

Comment:
I would add this note at the end of the section to focus first on the inventory part, then open to other folders/repo.

Reply:
Done.